x-pack/plugins/cross_cluster_replication/public/app/constants/form_schemas.js

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import Joi from 'joi';
+import { INDEX_ILLEGAL_CHARACTERS_VISIBLE } from 'ui/indices';
+
+import { indexNameValidator } from '../services/input_validation';
+
+const indexNameIllegalCharacters = INDEX_ILLEGAL_CHARACTERS_VISIBLE.join(' ');
+
+/* eslint-disable max-len */
+export const follwerIndexFormSchema = {
+  name: {
+    label: i18n.translate('xpack.crossClusterReplication.followerIndexForm.sectionFollowerIndexNameTitle', {
+      defaultMessage: 'Name'
+    }),
+    description: i18n.translate('xpack.crossClusterReplication.followerIndexForm.sectionFollowerIndexNameDescription', {
+      defaultMessage: 'A name for the follower index.'
+    }),
+    helpText: i18n.translate('xpack.crossClusterReplication.followerIndexForm.indexNameHelpLabel', {
+      defaultMessage: 'Spaces and the characters {characterList} are not allowed.',
+      values: { characterList: <strong>{indexNameIllegalCharacters}</strong> }
```

Comment:
I think this needs to be `{indexNameIllegalCharacters.join(' ')}`, or similar. currently shows as `[object Object]`:

![image](https://user-images.githubusercontent.com/1965714/51006703-57905600-14f9-11e9-85f6-41e9611bab7b.png)

Reply:
Yep, thanks for the catch!